### PR TITLE
Make `version` optional for dependencies

### PR DESCRIPTION
**Pull request by felipecrs**
This is a follow-up of https://github.com/SchemaStore/schemastore/pull/1815, where I stated my reasonings.

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
Sorry. I didn’t see that this actually refers to an open pull request that has not been merged.

In the future can you please link upstream PRs as well as mark PRs as DO NOT MERGE until the upstream PR has been merged first? Thanks.

helm/helm#10096

Reverting for now.

**Comment by felipecrs** [comment]
> In the future can you please link upstream PRs as well as mark PRs as DO NOT MERGE until the upstream PR has been merged first? Thanks.

For sure. My bad!

**Pull request merged**
